modules/ROOT/pages/deployment/general/general-info.adoc

```diff
@@ -356,19 +356,22 @@ log:
 You can easily access Infinite Scale via ownCloud Web with minimal configuration needs. Without going into too much detail, you need to provide the following two environment variables. See also the section about xref:handling-certificates[] and xref:demo-users-and-groups[].
 
 OCIS_URL::
-Expects a URL including _protocol_, _host_ and optionally _port_ to simplify configuring all the different services. Other service environment variables also using an URL still take precedence if set, but will fall back to this URL if not set.
-+
-NOTE: If you need to access Infinite Scale running on a VM or a remote machine via a host name other than localhost or in a container, you need to configure the host name with `OCIS_URL`. The same applies if you are not using host names but an IP address (e.g. 192.168.178.25) instead.
-+
-NOTE: By default, Infinite Scale enforces https for web and client access. If necessary, this can be changed to http, which is not recommended for production. For details see xref:deployment/services/tls.adoc#tls-for-the-http-frontend[TLS for the HTTP Frontend] and xref:deployment/services/s-list/proxy.adoc[Proxy Service Configuration].
 +
-include::partial$multi-location/idm-https-reverse-proxy.adoc[]
+--
+Expects a URL including _protocol_, _host_ and optionally _port_ to simplify configuring all the different services. Other service environment variables also using an URL still take precedence if set, but will fall back to this URL if not set.
+
+NOTE: If you need to access Infinite Scale running on a VM or a remote machine via a host name other than localhost or in a container, you *must* configure the host name with `OCIS_URL`. The same applies if you are not using host names but an IP address (e.g. 192.168.178.25) instead.
 
+NOTE: By default, Infinite Scale enforces https for web and client access. If necessary, this can be changed in particulart setups to http, which is *not* recommended for production. For details see xref:deployment/services/tls.adoc#tls-for-the-http-frontend[TLS for the HTTP Frontend] and xref:deployment/services/s-list/proxy.adoc#special-settings[Proxy Service Configuration].
```

Comment:
```suggestion
NOTE: By default, Infinite Scale enforces https for web and client access. If necessary, this can be changed in particular setups to http, which is *not* recommended for production. For details see xref:deployment/services/tls.adoc#tls-for-the-http-frontend[TLS for the HTTP Frontend] and xref:deployment/services/s-list/proxy.adoc#special-settings[Proxy Service Configuration].
```